templates/bot/js/notification-function-base/README.md

```diff
@@ -1,3 +1,88 @@
 # Notification Bot
 
-For how to use the notification bot, please refer to [this document](https://aka.ms/teamsfx-notification).
\ No newline at end of file
+A notifiction bot is an app that proactively sends messages in Teams channel / group chat / personal chat.
+
+For example, once triggered, it sends text, card, or other message(s) to Teams:
+
+![Notification Message in Teams](images/notification-message.png)
+
+## Get Started
+
+Before run this app locally, make sure you have prepared these prerequisites:
+
+- Node.js (Recommended version is 14)
+- An [M365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+
+Then, you can quickly start local debugging via `F5` in VSCode. Select `Debug (Edge)` or `Debug (Chrome)` debug option of your preferred browser.
+
+## Develop
+
+This new project folder structure looks like:
+
+| Folder | Contents |
+| - | - |
+| `.fx` | Project level settings and configurations |
```

Comment:
Some of the files in `.fx` are about environment information, not project-level configs.

Reply:
Added "environment information" as well